bundles/framework/coordinatetool/plugin/CoordinatePluginHandler.js

```diff
@@ -0,0 +1,626 @@
+import { StateHandler, controllerMixin, Messaging } from 'oskari-ui/util';
+import { showCoordinatePopup } from './CoordinatePopup';
+import { PLACEMENTS } from 'oskari-ui/components/window';
+import { getTransformedCoordinatesFromServer, transformCoordinates, formatDegrees } from './helper';
```

Comment:
getTransformedCoordinatesFromServer is implemented in this file and the imported one isn't used?

Reply:
This is called from a function in handler with identical name, which is a bit confusing. Should probably change it to something else.